src/handlers/deletelink.ts

```diff
@@ -42,7 +42,10 @@ export default async (formData: FormData, text: string) => {
 		const userName = formData.get('user_name');
 
 		if (deleteKW === "delete" && path && key === SECRET_KEY) {
-			await deleteOldUrl(path);
+			const response = await deleteOldUrl(path);
```

Comment:
Ahh, I see what you're getting, but I don't think nimbus throws a non 2xx. The error is caught and the error message is just sent as a normal response (line 59).

For deletelink, I would need to use if-else since we wouldn't want to tell the user that the link was deleted, and I thought it would be cleaner to just utilize the the catch behavior.

Reply:
Nimbus just throws it internally right, I think one `if (status === 2xx) slack()` line is fine for both